doc/index.rst

```diff
@@ -94,10 +82,9 @@ Please refer to the TSSW JIRA Workflow diagram above.
     * Ensure it is assigned correctly (Assignee, Component , etc)
     
       * Ensure the ticket is assigned to an actual, currently employed at LSST, person.
-      * Each ticket is assigned a JIRA Component.
-        If an Assignee is not designated, the JIRA Component will determine the assignee.
-    
-        * The component to assignee designation can be found `here <https://jira.lsstcorp.org/projects/TSS?selectedItem=com.atlassian.jira.jira-projects-plugin:components-page>`_.
+      * Each ticket is assigned a JIRA Epic.
+      * A label with the name of the component should be added to the ticket.
```

Comment:
probably should also say, each ticket should be assigned a component.  The Component should start with "ts_".  Not sure how relevant this is anymore...